src/stream/src/executor/mview/materialize.rs

```diff
@@ -207,6 +233,61 @@ impl<S: StateStore, SD: ValueRowSerde> MaterializeExecutor<S, SD> {
             }
         }
     }
+
+    fn mutation_might_affect_dispatcher(
+        mutation: Option<Arc<Mutation>>,
+        actor_id: ActorId,
+    ) -> bool {
+        let Some(mutation) = mutation.as_deref() else {
+            return false;
+        };
+        match mutation {
+            Mutation::Add(AddMutation { adds, .. }) => {
```

Comment:
I guess `Mutation::Add` is the only possible way to turn the dispatchers from zero to non-zero.

Reply:
Do you know what is the difference between `AddAndUpdate` and `Add`? And in which circumstances, `AddAndUpdate` would be used?

Reply:
It's for sink-into-table. So the upstream job must be a sink instead of materialized view. May check `add_dispatchers` call in `pre_mutate_dispatchers`.

https://github.com/risingwavelabs/risingwave/blob/023a7fdba716d229614d266d884b3262c4370f1f/src/stream/src/executor/dispatch.rs#L236-L247

BTW, this sounds fragile. Would you mind adding some comments?

Reply:
Just found that `AddAndUpdate` matters. We can use `sink into table` from a table.